pep8speaks/utils.py

```diff
@@ -28,8 +28,14 @@ def query_request(query=None, method="GET", **kwargs):
         "headers": {"Authorization": f"Bearer {GITHUB_TOKEN}"}
     }
 
-    for kw in kwargs:
-        request_kwargs["headers"].update(kwargs[kw])
```

Comment:
The old code had a bug where non-header parameters were being placed under the `headers` key, instead of at the top level

e.g.
```
# Incorrect
{
  "headers": {
    "a": 1,
    "json": {...},
  },
}
```
vs.
```
# Correct
{
  "headers": {"a": 1},
  "json": {...},
}
```